### DISCUSSION TITLE
Jasco 55258-zw4002 slider steps

### DISCUSSION THREAD

**Discussion by gene-vigelis** [zwave-js/zwave-js#4655] [Ideas] [answered]
Is there any way to update the configuration file for the Jasco 55258 3 speed switch so that the slider move in increments of 33.333% in the same way the Insteon Fanlinc Fan entity does?

**Comment by AlCalzone** [comment, marked as answer]
Since there is almost no information to go by, I'll have to guess here.

You're probably referring to a multilevel switch value, which by definition goes from 0 to 99. The Z-Wave specifications don't have a way to query if a device only supports certain steps, and we don't have a way to force that behavior.

If you are limited to certain steps in HAs UI for other devices, this is something that's done in the HA code base and not the driver.
▶️ https://github.com/home-assistant/core

**Comment by gene-vigelis** [comment]
FYI. This is the control on a fan.fanlinc item that only allows 0, 33, 66
and 100%. The slider will not stop anywhere in between:

[image: image.png]

[image: image.png]
